src/overlays/actors/ovl_En_Mag/z_en_mag.h

```diff
@@ -8,9 +8,51 @@ struct EnMag;
 
 typedef struct EnMag {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0xE1DC];
+    /* 0x014C */ char unk_14C[0x0004];
+    /* 0x0150 */ u8 unk_150[0xE188]; // kanfont struct/buffer
+    /* 0xE2D8 */ s16 effectFadeInState;
+    /* 0xE2DA */ s16 effectFadeInTimer;
+    /* 0xE2DC */ s16 globalState;
+    /* 0xE2E0 */ f32 effectPrimLodFrac;
+    /* 0xE2E4 */ f32 effectPrimColor[3];
+    /* 0xE2F0 */ f32 effectEnvColor[3];
+    /* 0xE2FC */ f32 effectAlpha;
+    /* 0xE300 */ f32 mainAlpha;
+    /* 0xE304 */ f32 subAlpha;
+    /* 0xE308 */ f32 copyrightAlpha;
+    /* 0xE30C */ s16 unk_E30C;
+    /* 0xE30E */ s16 effectScroll;
+    /* 0xE310 */ char unk_E310[0x0002];
+    /* 0xE312 */ s16 copyrightAlphaStep;
+    /* 0xE314 */ s16 fadeOutAlphaStep;
+    /* 0xE316 */ s16 unk_E316;
+    /* 0xE318 */ s16 unk_E318;
+    /* 0xE31C */ s32 unk_E31C;
+    /* 0xE320 */ s32 unk_E320;
+    /* 0xE324 */ char unk_E324[0x0004];
 } EnMag; // size = 0xE328
 
 extern const ActorInit En_Mag_InitVars;
 
+typedef enum {
```

Comment:
Since the `globalState` is part of the actor struct exposed in the header, I think it makes sense to expose the enum alongside it, since you would want it when accessing the struct member.
The custom gfx macro is debatable but I feel like it belongs in the header as well, since it's not specific to the actor and probably supposed to be part of some gbi related header. It doesn't really matter either way since nothing else will probably ever include this en_mag header, so it's kinda w/e.

Reply:
I also dont feel too fond of globalState but if theres a possibility of naming it something better later anyway, then its all good